### PR TITLE
2.4 Release Prep

### PR DESCRIPTION
**Pull request by sirlensalot**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # `chainweb-node` Changelog
 
+## 2.4 (2021-01-11)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2020-01-14T00:00:00Z**. Node administrators must upgrade to this version
```
Comment (mightybyte) [inline CHANGELOG.md:6]: This should be 2021.

File: node/ChainwebNode.hs [hunk 2]
```diff
@@ -494,7 +494,7 @@ pkgInfoScopes =
 -- KILLSWITCH for version 2.3
 --
 killSwitchDate :: Maybe String
-killSwitchDate = Just "2021-01-14T00:00:00Z"
+killSwitchDate = Just "2021-02-25T00:00:00Z"
```
Comment (mightybyte) [inline node/ChainwebNode.hs:497]: Looks good

File: src/Chainweb/Version.hs [hunk 3]
```diff
@@ -852,8 +852,8 @@ enablePactEvents _ bh = bh >= 2
 
 -- | Bridge support: ETH and event SPV.
 enableSPVBridge :: ChainwebVersion -> BlockHeight -> Bool
-enableSPVBridge Mainnet01 = (>= 2000000) -- TODOTODOTODOTODOTODOTODO
-enableSPVBridge Testnet04 = (>= 1900000) -- TODOTODOTODOTODOTODOTODO
+enableSPVBridge Mainnet01 = (>= 1_275_000) -- 2021-01-14T16:35:58
```
Comment (mightybyte) [inline src/Chainweb/Version.hs:855]: Passes my cursory general estimate check.

**Review by mightybyte — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+before that date.
+
+This version will stop working on **2021-02-25T00:00:00Z**.
+
+ *  Ethereum bridge support
+   *  add endpoint for creating eth-receipt-proofs (#1181)
+   *  ethereum receipt SPV (#1179)
+ *  Improve TXOUT SPV to include events (#1178)
+ *  Pact replay bug fix (#1172,#1169,#1168)
+ *  run brew update before installing rocksdb (#1171)
```
Comment (larskuhtz) [inline CHANGELOG.md:16]: This is only an Github Actions CI fix.
```suggestion
```

File: src/Chainweb/Version.hs (hunk 3 above)
Comment (larskuhtz) [reply src/Chainweb/Version.hs:855]: 👍  This is what I get:

Mainnet: "2021-01-14 16:35:25 UTC"
Tetsnet: "2021-01-14 17:09:56 UTC"

**Review by mightybyte — APPROVED** [review]

**Review by larskuhtz — APPROVED** [review]

**Pull request merged**